templates/team.html

```diff
@@ -23,15 +23,23 @@ <h1>Staff</h1>
         </a>
       </footer>
       <p class="justify">
-        Daaimah, a queer, black, disabled individual and primary caregiver,
-        entered the tech industry facing numerous barriers. After completing the
-        Techtonica program in 2019, she worked as a software engineer at
-        SurveyMonkey, tech lead at Hack The Hood, and technical career counselor
-        with Code Tenderloin. She later became the Software Training Engineering
-        Manager at Techtonica, advocating for accommodations and training access
-        for underrepresented groups in tech. Daaimah grew up and lives in the
-        Bay Area. She enjoys hiking with her family and learning about science
-        and anthropology.
+        As a queer, black, disabled, primary caregiver, Daaimah faced many
+        barriers simply entering the tech industry and when getting into her
+        first technical role; she discovered it was a whole new beast compared
+        to her past work in the grassroots nonprofit world: writing grants,
+        managing programs, supporting equity and inclusion through public policy
+        education, and advising persons with disabilities on careers. Following
+        completion of the Techtonica program in 2019, she worked at SurveyMonkey
+        as a software engineer, tech lead at Hack The Hood, and technical career
+        counselor with Code Tenderloin. Wanting to pay it forward, Daaimah
+        volunteered as a graduate board member and mentor program manager at
+        Techtonica until she was hired as the Software Training Engineering
+        Manager at Techtonica, where she now utilizes the skill sets in her
+        arsenal to advocate for mindful accommodations, training access, and new
+        pathways to entry for underrepresented intersectionalities in the tech
+        space. She grew up and still resides in the Bay Area. For kicks, she
+        enjoys hitting the trails with her two children and two dogs, as well as
+        learning about science and anthropology.
```

Comment:
Please see suggestions for what is currently live:
```suggestion
        Daaimah, a queer, black, disabled individual and primary caregiver, entered the tech industry facing numerous barriers. After completing the Techtonica program in 2019, she worked as a software engineer at SurveyMonkey, tech lead at Hack The Hood, and technical career counselor with Code Tenderloin. She later became the Software Training Engineering Manager at Techtonica, advocating for accommodations and training access for underrepresented groups in tech. Daaimah grew up and lives in the Bay Area. She enjoys hiking with her family and learning about science and anthropology.
```

Reply:
Resolved in new rebase!